src/dbnode/storage/repair_test.go

```diff
@@ -558,6 +561,21 @@ func TestDatabaseShardRepairerRepairMultiSession(t *testing.T) {
 		{Host: hosts[1], Metadata: inputBlocks[1].Metadata},
 	}
 	require.Equal(t, expected, currBlock.Metadata())
+
+	// Validate the expected metrics were emitted
+	scopeSnapshot := scope.Snapshot()
+	countersSnapshot := scopeSnapshot.Counters()
+	gaugesSnapshot := scopeSnapshot.Gauges()
+	require.Equal(t, int64(2),
+		countersSnapshot["repair.series+namespace=testNamespace,resultType=total,shard=0"].Value())
```

Comment:
There's a helper you can use to generate this key btw: `tally.KeyForPrefixedStringMap`

Reply:
Fixed.